torchci/test/mergeCommands.test.ts

```diff
@@ -607,7 +607,7 @@ describe("merge-bot", () => {
   test("merge fail because mutually exclusive options", async () => {
     const event = requireDeepCopy("./fixtures/pull_request_comment.json");
 
-    event.payload.comment.body = "@pytorchbot merge -ic -f '[MINOR] Fix lint'";
+    event.payload.comment.body = "@pytorchbot merge -i -f '[MINOR] Fix lint'";
```

Comment:
This would fail, but for a different reason: `-f` does not accept an argument. So ether remove the string or add `-m` option.
```suggestion
    event.payload.comment.body = "@pytorchbot merge -i -f -m 'Twas brillig, and the slithy toves'";
```